test-files/cmd/slow/ConflateCmdStatsGenericRiversTest.sh.stdout

```diff
@@ -10,13 +10,13 @@ Total Unconflatable Features	0	0	0
 Percentage of Total Features Unconflatable	0	0	0	
 Number of Match Creators	1	1	1	
 Features Conflatable by: hoot::ScriptMatchCreator,LinearWaterway.js	14	15	0	
-Total Conflated Features	0	0	17	
-Percentage of Total Features Conflated	0	0	16.19047619047619	
+Total Conflated Features	0	0	19	
+Percentage of Total Features Conflated	0	0	18.09523809523809	
```

Comment:
Since more ways have the right status, the stats are better now that two more ways were conflated.